src/components/POLine/POLineForm.js

```diff
@@ -281,7 +282,7 @@ class POLineForm extends Component {
                       label={<FormattedMessage id="ui-orders.line.accordion.location" />}
                       id={ACCORDION_ID.location}
                     >
-                      <LocationForm {...this.props} />
+                      <LocationForm {...this.props} orderFormat={orderFormat} />
```

Comment:
You can pass only:
`orderFormat={orderFormat}`
`parentResources={parentResources}`

Reply:
It's not an issue you've introduced, but it would be great to fix, thanks!

Reply:
fixed